### PR TITLE
Fix Bug 1097785 - Legal-docs-based pages should not rely on active tags in .lang files

### PR DESCRIPTION
**Pull request by kyoshino**

### CODE REVIEW

File: bedrock/legal_docs/views.py [hunk 1]
```diff
@@ -25,10 +26,16 @@ def load_legal_doc(doc_name, locale):
 
     :param doc_name: name of the legal doc folder
     :param locale: preferred language version of the doc
-    :return: string content of the file or None
+    :return: dict containing string content of the file (or None), a boolean
+             value indicating whether the file is localized into the specified
+             locale, and a dict of all available locales for that document
     """
-    source = path.join(LEGAL_DOCS_PATH, doc_name, locale + '.md')
+    dir = path.join(LEGAL_DOCS_PATH, doc_name)
```
Comment (pmclanahan) [inline bedrock/legal_docs/views.py]: This is shaddowing the builtin function `dir`. Make it a more explicit name to avoid confusion.

**Comment by pmclanahan** [comment]
Apart from that one thing the code looks good. I'll try to run it locally latter and we should be good after that. Thanks again, and apologies for the review delay.

File: bedrock/legal_docs/views.py (hunk 1 above)
Comment (kyoshino) [inline bedrock/legal_docs/views.py]: Changed the variable name `dir` to `source_dir`.

File: lib/l10n_utils/__init__.py [hunk 2]
```diff
@@ -41,7 +41,8 @@ def render(request, template, context=None, **kwargs):
     context['langfile'] = get_lang_path(template)
 
     # Get the available translation list of the current page
-    context['translations'] = translations_for_template(template)
+    context['translations'] = context.get('translations', {})
```
Comment (pmclanahan) [inline lib/l10n_utils/__init__.py]: NIT: I think `context.setdefault('translations', {})` is more appropriate here.

**Comment by pmclanahan** [comment]
Works well. Nice work Kohei! Let me know if you'd like to change that NIT I mentioned, otherwise I'll merge later today. Thanks again!

**Comment by pmclanahan** [comment]
r+

File: lib/l10n_utils/__init__.py (hunk 2 above)
Comment (kyoshino) [inline lib/l10n_utils/__init__.py]: Done replacing `context.get` with `context.setdefault`!

File: lib/l10n_utils/__init__.py [hunk 3]
```diff
@@ -41,7 +41,8 @@ def render(request, template, context=None, **kwargs):
     context['langfile'] = get_lang_path(template)
 
     # Get the available translation list of the current page
-    context['translations'] = translations_for_template(template)
+    context['translations'] = context.setdefault('translations', {})
```
Comment (pmclanahan) [inline lib/l10n_utils/__init__.py]: Sorry, I'm sure I wasn't clear. It should be:

```python
context.setdefault('translations', {})
context['translations'].update(translations_for_template(template))
```

Comment (kyoshino) [inline lib/l10n_utils/__init__.py]: Ah, thank you, I should have read it carefully. Fixed.

**Comment by pmclanahan** [comment]
Nice! Thanks @kyoshino.

**Pull request merged**